### PR TITLE
chore(travis): automate lerna travis release

### PR DESCRIPTION
**Pull request by nellyk**
Automate release of lerna packages:

This would follow this process:

1. Run `npm run lerna:version` locally from your release branch. This would push your release changes(changelog and tags) to the branch on github.
fix(transaction/details): missing quantity field
2. Create a pull request from your branch to the `master` branch with your changes.
3. Once the changes are approved ensure you merge the changes and provide the commit message for the merge as`chore(release): <semantic-release-version>`
4. Once this is merged to master a deployment would be initiated on the master branch and all packages that changed would be released to npm.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,11 +1,24 @@
...
+  include:
+    # Define the release stage that publishes lerna to npm
+    - stage: release
+      deploy:
+          on:
+            branch: master
+          provider: script
+          skip_cleanup: true
+          script:
+            - npm run lerna:deploy -- "${TRAVIS_COMMIT_MESSAGE}
```
Comment (mtomcal) [inline .travis.yml]:
```suggestion
            - npm run lerna:deploy -- "${TRAVIS_COMMIT_MESSAGE}"
```

File: package.json [hunk 2]
```diff
@@ -13,7 +13,10 @@
     "test": "NODE_ENV=development jest",
     "test:git-history": "commitlint --from origin/master --to HEAD",
     "test:lockfile": "node lockFileLint.js",
-    "posttest": "npm run lint && npm run test:git-history && npm run test:lockfile"
+    "posttest": "npm run lint && npm run test:git-history && npm run test:lockfile",
+    "lerna:version": "lerna version",
+    "lerna:publish": "lerna publish from-package --yes",
+    "lerna:deploy":"node lernaDeploy.js"
```
Comment (mtomcal) [inline package.json]:
```suggestion
    "lerna:deploy": "node lernaDeploy.js"
```

**Comment by mtomcal** [comment]
Why 
`chore(release) : <semantic-release-version>`
and not
`chore(release): <semantic-release-version>`

File: package.json (hunk 2 above)
Comment (JamesSingleton) [inline package.json]: There should be a space between `:` and `"node lernaDeploy.js"`

Comment (nellyk) [reply package.json]: Thank you updated a9da5b2

Comment (nellyk) [reply package.json]: updated a9da5b2

**Pull request merged**